workspaces/liferay-dlfoldertemplate-workspace/client-extensions/liferay-dlfoldertemplate-custom-element/src/components/template-list/TemplateList.js

```diff
@@ -106,47 +106,26 @@ const TemplateList = () => {
 	};
 
 	const confirmDeleteItemModal = (template) => {
+
 		const deleteTemplate = async () => {
+
 			setIsDeletingLoading(true);
 
 			await deleteFolderTemplateInformation(template.id);
 
 			setIsDeletingLoading(false);
 
 			reload();
-
-			modalState.onClose();
 		};
 
-		dispatchModal({
-			payload: {
-				body:
-					'Deleting an Template also removes its entries. This action is permanent and cannot be undone.',
-				center: true,
-				footer: [
-					,
-					,
-					<ClayButton
-						disabled={isDeletingLoading}
-						displayType="danger"
-						onClick={() => {
-							deleteTemplate();
-						}}
-					>
-						{isDeletingLoading && (
-							<ClayLoadingIndicator
-								displayType="danger"
-								size="sm"
-							/>
-						)}
-						{'Delete'}
-					</ClayButton>,
-				],
-				header: 'Delete Folder Template',
-				size: 'lg',
-				status: 'danger',
+		Liferay.Util.openConfirmModal({
+			message:
+				'Deleting an Template also removes its entries. This action is permanent and cannot be undone.',
```

Comment:
Wordsmithing:

"Deleting a template also removes its entries. This action is permanent and cannot be undone."